### PR TITLE
Add dlist.gg

### PR DESCRIPTION
**Pull request by advaith1**
https://discordlist.gg

### CODE REVIEW

File: data/lists/discordlist.gg.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "id": "discordlist.gg",
+    "added": 1542972675,
```
Comment (MattIPv4) [inline data/lists/discordlist.gg.json:3]: This timestamp doesn't seem correct? It should be "now" to represent when the list is added to BotBlock

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: data/lists/discordlist.gg.json (hunk 1 above)
Comment (advaith1) [reply data/lists/discordlist.gg.json:3]: oops didnt see that

**Review by MattIPv4 — APPROVED** [review]
LGTM, ty

**Comment by CircuitSacul** [comment]
Gives a 405: Method not allowed.

I assume this is because discordlist wants PUT requests, not POST requsts.

Screen Shot 2022-07-15 at 10 13 57 PM

**Comment by renhiyama** [comment]
I love how everyone uses different methods and "methods"

**Comment by MattIPv4** [comment]
@CircuitSacul Please could open an issue over in the api-worker for this? I suspect we'll need to add a new api_method property or similar and add support for that in the worker.

**Pull request merged**